src/main/java/mate/academy/rickandmorty/controller/CharactersController.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.rickandmorty.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.model.Character;
+import mate.academy.rickandmorty.service.ClientService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/api/characters")
+@RequiredArgsConstructor
+@Tag(name = "Api for searching characters of Rick and Morty series",
+        description = "Helps to find characters by name or provides a random character")
+public class CharactersController {
```

Comment:
lets use singular form 
```suggestion
public class CharacterController {
```